src/libultra/flash/osFlash.c

```diff
@@ -0,0 +1,325 @@
+#include "ultra64.h"
+#include "global.h"
+
+static s32 D_801FD040[4];
```

Comment:
It looks like an array of fram device info.
```suggestion
static s32 framDeviceInfo[4];
```